plans/jenkins/jenkins.yaml

```diff
@@ -141,7 +142,7 @@ jenkins:
       extension: Ansible
       source:
         location: "git+https://github.com/lacework-dev/detc-resources.git"
-        subdir: "apps/jenkins/ansible"
+        subdir: "apps/jenkins/ansible/"
```

Comment:
Does this trailing slash matter?

Reply:
Fixed, didn't mean to make that change....